src/dependency.rs

```diff
@@ -95,7 +105,13 @@ impl Dependency {
                 if self.optional {
                     data.get_or_insert("optional", optional);
                 }
-
+                if !self.features.is_empty() {
+                    let mut to_array = toml_edit::Array::default();
```

Comment:
`toml_edit::Value` [implements](https://docs.rs/toml_edit/0.1.1/src/toml_edit/formatted.rs.html#202-215) `FromIterator`, so we can simplify this:
```rust
use std::iter::FromIterator;

let features = toml_edit::Value::from_iter(self.features.iter().cloned());
data.get_or_insert("features", features);
```

but we need to specify feature list only if user specified it.